frappe_pwa/frappe_pwa/doctype/web_app_manifest/web_app_manifest.py

```diff
@@ -7,11 +7,22 @@
 
 from frappe.model.document import Document
 
+
 class WebAppManifest(Document):
+    def on_update(self):
+        """clear cache"""
+        frappe.clear_cache(user='Guest')
+
+        from frappe.website.render import clear_cache
+        clear_cache()
 
-	def on_update(self):
-		"""clear cache"""
-		frappe.clear_cache(user = 'Guest')
 
-		from frappe.website.render import clear_cache
-		clear_cache()
+@frappe.whitelist()
+def configure_pwa():
+    manifest = '''<link href="/assets/frappe_pwa/manifest.json" rel="manifest">'''
```

Comment:
Correct me if I'm wrong, but the since the `manifest.json` is dynamically generated using Jinja, the correct path is `/manifest.json`.
```suggestion
    manifest = '''<link href="/manifest.json" rel="manifest">'''
```